app/components/satis/menu/component_controller.js

```diff
@@ -37,10 +37,10 @@ export default class extends ApplicationController {
       })
     }
 
-    if(this.hasToggleugTarget) {
-      let groupByData = JSON.parse(this.toggleugTarget.dataset.show_params)
-      if(this.toggleugTarget.id == "group_by_"+groupByData.current_view+"_"+groupByData.group_by_column) {
-        this.toggleugTarget.classList.toggle("hidden")
+    if(this.hasClearTarget) {
+      let data = JSON.parse(this.clearTarget.dataset.show_params)
+      if (this.clearTarget.id == "group_" + data.current_view + "_" + data.group_by_column) {
```

Comment:
Much better, can we add an attribute/value to the menu-item/component for which we check the id? As in data-clear-id="group_viewid_column" and set that from action-table?